sdk-extensions/tracing-incubator/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ extra["moduleName"] = "io.opentelemetry.sdk.extension.trace.incubator"
 dependencies {
     api(project(":api:all"))
     api(project(":sdk:all"))
+    api(project(":sdk:metrics"))
```

Comment:
I think we want to leave `metrics` as an implementation detail for now, like we have in the core sdk trace module.

Reply:
👍